### DISCUSSION TITLE
[Keyball61]フラックス除去後からキーが反応しなくなった

### DISCUSSION THREAD

**Discussion by sabaemon1104** [Yowkees/keyball#685] [Q&A]
お世話になっております。
keyball61を初めて作成しています（はんだ付け等も初めてです）。

Keyball61の完成直後はすべてのキーが問題なく作動していたのですが、
フラックス除去液を用いてフラックスを除去した後、キーマップを変更を行っていたところ、
右手トラックボール近くにある添付赤丸のキー（SW20, SW27だと思います）が反応しなくなっていました。
添付黄色のダイオードと合わせて何度か加熱しなおしてみたのですが、状況変わりません。
原因・対応について、お知恵をお借りできませんでしょうか。

![PXL_20241104_140157529](https://github.com/user-attachments/assets/2d5dcb2e-2a95-4684-8a7f-fd4f77801e1c)
![PXL_20241104_141608965](https://github.com/user-attachments/assets/d58ff06f-66e8-4b54-a3fb-e7cfc9b63ae5)
![PXL_20241104_141900465 MP](https://github.com/user-attachments/assets/09a52fa6-1959-4e79-bf55-a8f63a6fa398)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。症状了解しました。

この２キーだけということは、１列全部とかではないので、このキー周辺に問題があると思います。
下記画像の緑点どうし、青点どうしは導通していますか？
またオレンジ点３点も導通しているでしょうか？

導通していない場合、半田付け作業中などに基板のパターンが傷ついた可能性もありますので、エナメル線等でショートしてください。

![61R2keysfix](https://github.com/user-attachments/assets/48b9fbbd-2d75-4976-967c-39ca2505b4e2)

お手数ですがご確認お願いいたします。

**Reply by sabaemon1104** [reply]
ご確認ありがとうございます！
オレンジの導通が確認できなかったため、オレンジ点の上と真ん中の2点をショートさせたところ、
どちらのキーも反応するようになりました！
おかげさまで、全てのキーが正常に反応するようになりました。
ありがとうございました！
